### DISCUSSION TITLE
fscrawler for arm64 rpi

### DISCUSSION THREAD

**Discussion by ludoblockdevice** [dadoonet/fscrawler#1134] [Q&A]
Hello,

Do you have any plans supporting arm64 architecture ? 
Could you please share me some information about such process ? Is it doable with common compiling tools ?

Thank you !

**Comment by dadoonet** [comment]
I don't have plans like this. I'm just wondering if there's anything to do on my side actually to support it.
FSCrawler runs with a JVM. So I guess you "just" need to install a JVM for arm64. 

Did you try:

```sh
sudo apt update
sudo apt install openjdk-15-jre-headless
```

And then run FSCrawler with that JVM?
